.github/actions/translation-tracker/README.md

```diff
@@ -0,0 +1,266 @@
+# p5.js Translation Tracker
+
+A comprehensive GitHub Action and command-line tool to track translation status across multiple languages in the p5.js website examples.
+
+## 🚀 Overview
+
+This tool helps maintain translation consistency by:
+- **Week 1**: Basic file-based change detection using Git
+- **Week 2**: GitHub API integration for accurate commit tracking and automated issue creation
+- **Week 3**: Multi-language support with single issues per file and manual scanning capabilities
+
+## 📋 Features
+
+### ✅ Week 1 Features (File-based tracking)
+- Git-based change detection for English example files
+- File modification time comparison
+- Support for all languages: Spanish (es), Hindi (hi), Korean (ko), Chinese Simplified (zh-Hans)
+- Local testing capabilities
+
+### ✅ Week 2 Features (GitHub API integration)
+- Real commit-based comparison using GitHub API
+- Automated GitHub issue creation for outdated translations
+- Enhanced issue templates with helpful links and timelines
+- Branch detection (auto-detects current branch)
+- Backward compatibility with Week 1
+
+### ✅ Week 3 Features (Multi-language & refinement)
+- **Single issue per file**: Creates one issue covering all affected languages instead of separate issues per language
+- **Enhanced labeling**: Uses "needs translation" base label + specific language labels (e.g., "lang-es", "lang-ko")
```

Comment:
Same here - please reduce the README content to be shorter (and in that way more maintainable)